### ISSUE TITLE
[Screen reader - Make code - Help > Tour]: Screen reader does not announce the dialog information which appears by invoking the 'Tour' menu item.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#5489]
### User Experience:
Screen reader dependent people will get impacted if screen reader does not announce dialog information by invoking the 'Tour' menu item. Due to which the end user will not be able to know the type of revealed content and cannot navigate it efficiently.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL : https://makecode.microbit.org/. 
2. Home page will open. Turn on the screen reader (JAWS).
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Help" option present on the header and invoke it.
6. Select 'Tour' menu item and observe whether screen reader announces the dialog information or not.

### Actual Result:
**Issue:** Screen reader does not announce the dialog information which appears by invoking the 'Tour' menu item.

**Observation:**
- Screen reader remains silent by invoking the 'Tour' menu item.
- **Narrator** and **NVDA** remains silent by invoking the 'Tour' menu item.

### Expected Result:
Screen reader should announce the dialog information which appears by invoking the 'Tour' menu item.

**Example:** By invoking the 'Tour' menu item, the screen reader should announce as 'Dialog - welcome - New here? Take a tour of the editor!'

### MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
Yes (Refer the attachment)

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- URL: https://makecode.microbit.org/.

https://github.com/microsoft/pxt-microbit/assets/158465724/ef0fabb8-d7c0-4cd3-9a82-18471c1e37b7

<img width="960" alt="MAS1 3 1_Screen reader does not announce the dialog information which appears by invoking the 'Tour' menu item" src="https://github.com/microsoft/pxt-microbit/assets/158465724/5c207ebf-b7a7-425f-bd4d-cc23c9c642cc">

**Comment by vroopea** [comment]
#Accessibility;#A11yMAS;#A11ySev3;#MAS1.3.1;#AILimited;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Feb2024;
